src/libs/Navigation/AppNavigator/MainDrawerNavigator.js

```diff
@@ -70,6 +78,17 @@ class MainDrawerNavigator extends Component {
         return true;
     }
 
+    trackAppStartTiming() {
+        if (!this.isFromCache) {
+            return;
+        }
+
+        // We only want to report the start time, if the sidebar was
+        // rendered with cached data, so we don't have any network
+        // times in that timing included.
+        Timing.end(CONST.TIMING.SIDEBAR_LOADED);
```

Comment:
The comment mentiones start time, but we're actually ending the timing here. If this is expected could we make this clearer in the comment please?

Reply:
yeah thats much better / clearer 👍